core/tests/protractor_desktop/preferences.js

```diff
@@ -144,11 +144,13 @@ describe('Preferences', function() {
       await users.createUser('lorem@preferences.com', 'loremPreferences');
       await users.login('lorem@preferences.com');
       await preferencesPage.get();
+      await waitFor.pageToFullyLoad();
```

Comment:
Could you add this to the end of the `preferencesPage.get()` function instead? I think that will make the code cleaner. Also, note that `waitFor.pageToFullyLoad();` really waits for a loading indicator to disappear, so please make sure that such an indicator is actually displayed when loading the preferences page

Reply:
Actually, I don't think this does anything. We already have a `waitFor.pageToFullyLoad()` at the end of the `preferencesPage.get()` function:

https://github.com/oppia/oppia/blob/010af888afca2bfa27050ebc68a5f9ec28430a39/core/tests/protractor_utils/PreferencesPage.js#L69-L72

Since this extra waitFor is redundant, can you leave the `preferences.js` file as-is?

Reply:
Done. I have removed autofocus from preference page currently. It will be done later in another PR.Thank you